Update globalnet related doc (#93)
submariner-io/submariner-website: src/content/quickstart/openshiftgn/_index.md

#### Join cluster-a and cluster-b to the broker

```bash

> subctl join --kubeconfig cluster-a/auth/kubeconfig broker-info.subm --clusterid west --globalnet-cidr 169.254.0.0/19

Reviewer: This should not be part of a quickstart, it's a feature that we have, but for very specific use cases. Specially not a "quick" start

Author: It was added as a precaution (and part of the comment https://github.com/submariner-io/submariner-website/pull/93#discussion_r420724783) to avoid facing race conditions that @manosnoam was facing last week.

Reviewer 2: @mangelajo As you know, we have a known issue where same globalCIDRs can get allocated to the clusters. This will be addressed as part of https://github.com/submariner-io/submariner-operator/issues/299

Until this is fixed, lets have the --globalnet-cidr included as part of Quickstart. We can remove it once the above use-case is addressed.

> subctl join --kubeconfig cluster-a/auth/kubeconfig broker-info.subm --clusterid west --globalnet-cidr 169.254.0.0/19

Reviewer: ```suggestion
subctl join --kubeconfig cluster-a/auth/kubeconfig broker-info.subm --clusterid west
```

```

```bash

> subctl join --kubeconfig cluster-b/auth/kubeconfig broker-info.subm --clusterid east --globalnet-cidr 169.254.32.0/19

Reviewer: ```suggestion
subctl join --kubeconfig cluster-b/auth/kubeconfig broker-info.subm --clusterid east
```

#### Join cluster-a and cluster-b to the broker

```bash
subctl join --kubecontext west --kubeconfig ./merged_kubeconfig broker-info.subm --clusterid west --broker-cluster-context west --globalnet-cidr 169.254.0.0/19
```

```bash

> subctl join --kubecontext east --kubeconfig ./merged_kubeconfig broker-info.subm --clusterid east --broker-cluster-context west --globanet-cidr 169.254.32.0/19

Reviewer 2: typo: globalnet-cidr

```

>  

Reviewer 3: ```suggestion

```

subctl join --kubecontext east --kubeconfig ./merged_kubeconfig broker-info.subm --clusterid east --broker-cluster-context west
```

> This will auto-assign Global CIDRs to the clusters. ```--globalnet-cidr``` flag can be used if Globalnet CIDRs need to be externally specified by the user.

Reviewer 4: ```suggestion
BY default this will auto-assign Global CIDRs to the clusters. The ```--globalnet-cidr``` flag can be used if Globalnet CIDRs need to be externally specified by the user.
```

> This will auto-assign Global CIDRs to the clusters. ```--globalnet-cidr``` flag can be used if Globalnet CIDRs need to be externally specified by the user.

Reviewer 3: ```suggestion
By default `subctl` will auto-assign global CIDRs to the clusters. The `--globalnet-cidr` flag can be used if the Globalnet CIDRs need to be assigned explicitly.
```

> subctl will check that the specified CIDRs do not overlap across clusters before joining the clusters.

Reviewer 2: Since this is a quick start guide, IMO its better to keep this short and move any additional information about GN to the corresponding GN architecture page. Reason being, if you see, as part of deploy-broker command above, we are not including any notes about default --globalnet-cidr-range, --globalnet-cluster-size etc. In GN architecture page, we can add some notes about the various globalnet parameters and how they work.

Author: I have added few lins about globalnet-cdr in the architecture page as well.

> subctl will check that the specified CIDRs are not overlapping and other sanity checks before deploying the clusters.

Reviewer 4: ```suggestion
subctl will check that the specified CIDRs do not overlap across clusters before joining the clusters.
```

Reviewer 2: Please apply the changes before marking the conversation as Resolved. As @tpantelis pointed out, we are not deploying the cluster, we are "joining" the clusters.

Also, like I commented earlier here- https://github.com/submariner-io/submariner-website/pull/93#pullrequestreview-405697504, the quickstart guide should be short and succinct. But this opinion could be subjective. If others are fine including this message here, it's Okay to me. 

We can also do the following, remove this description from quickstart and explicitly include **--globanet-cidr 169.254.0.0/19 for cluster-a** and **--globanet-cidr 169.254.32.0/19 for cluster-b** as part of join command above. This will be a solution to some of the race conditions we are seeing until the [issue](https://github.com/submariner-io/submariner-operator/issues/299) is resolved.

Author: > Please apply the changes before marking the conversation as Resolved. As @tpantelis pointed out, we are not deploying the cluster, we are "joining" the clusters.
> 
I missed the joining part. As can be seen other comments which need discussion are still open and not resolved.

> Also, like I commented earlier here- [#93 (review)](https://github.com/submariner-io/submariner-website/pull/93#pullrequestreview-405697504), the quickstart guide should be short and succinct. But this opinion could be subjective. If others are fine including this message here, it's Okay to me.
> 
> We can also do the following, remove this description from quickstart and explicitly include **--globanet-cidr 169.254.0.0/19 for cluster-a** and **--globanet-cidr 169.254.32.0/19 for cluster-b** as part of join command above. This will be a solution to some of the race conditions we are seeing until the [issue](https://github.com/submariner-io/submariner-operator/issues/299) is resolved.

I like adding the flags as part of join command. And a short description of it is anyways added to the architecture section.

> subctl will check that the specified CIDRs are not overlapping and other sanity checks before deploying the clusters.

Reviewer 2: Please apply the changes before marking the conversation as Resolved. As @tpantelis pointed out, we are not deploying the cluster, we are "joining" the clusters.

Also, like I commented earlier here- https://github.com/submariner-io/submariner-website/pull/93#pullrequestreview-405697504, the quickstart guide should be short and succinct. But this opinion could be subjective. If others are fine including this message here, it's Okay to me.

We can also do the following, remove this description from quickstart and explicitly include **--globanet-cidr 169.254.0.0/19 for cluster-a** and **--globanet-cidr 169.254.32.0/19 for cluster-b** as part of join command above. This will be a solution to some of the race conditions we are seeing until the issue is resolved.
